[2024-05-18T23:20:57.197+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T23:20:57.222+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Webscrape.Process_Send_Data manual__2024-05-18T23:20:54.361544+00:00 [queued]>
[2024-05-18T23:20:57.227+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Webscrape.Process_Send_Data manual__2024-05-18T23:20:54.361544+00:00 [queued]>
[2024-05-18T23:20:57.227+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-18T23:20:57.246+0000] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): Process_Send_Data> on 2024-05-18 23:20:54.361544+00:00
[2024-05-18T23:20:57.267+0000] {standard_task_runner.py:63} INFO - Started process 201 to run task
[2024-05-18T23:20:57.275+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Webscrape', 'Process_Send_Data', 'manual__2024-05-18T23:20:54.361544+00:00', '--job-id', '323', '--raw', '--subdir', 'DAGS_FOLDER/webscrapping.py', '--cfg-path', '/tmp/tmpxrdg4m43']
[2024-05-18T23:20:57.284+0000] {standard_task_runner.py:91} INFO - Job 323: Subtask Process_Send_Data
[2024-05-18T23:20:57.431+0000] {task_command.py:426} INFO - Running <TaskInstance: Webscrape.Process_Send_Data manual__2024-05-18T23:20:54.361544+00:00 [running]> on host c0373bd0174c
[2024-05-18T23:20:57.605+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='svc' AIRFLOW_CTX_DAG_ID='Webscrape' AIRFLOW_CTX_TASK_ID='Process_Send_Data' AIRFLOW_CTX_EXECUTION_DATE='2024-05-18T23:20:54.361544+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-18T23:20:54.361544+00:00'
[2024-05-18T23:20:57.611+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T23:20:57.655+0000] {base.py:84} INFO - Using connection ID 'spark_default' for task execution.
[2024-05-18T23:20:57.657+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --packages org.apache.spark:spark-sql-kafka-0-10_2.12:3.5.1,com.datastax.spark:spark-cassandra-connector_2.12:3.5.0,org.apache.kafka:kafka-clients:3.5.1,commons-logging:commons-logging:1.2 --name arrow-spark --deploy-mode client dags/includes/jobs/spark_consumer.py
[2024-05-18T23:20:57.855+0000] {spark_submit.py:571} INFO - /home/***/.local/lib/python3.11/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-05-18T23:21:00.670+0000] {spark_submit.py:571} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.11/site-packages/pyspark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2024-05-18T23:21:00.745+0000] {spark_submit.py:571} INFO - Ivy Default Cache set to: /home/***/.ivy2/cache
[2024-05-18T23:21:00.745+0000] {spark_submit.py:571} INFO - The jars for the packages stored in: /home/***/.ivy2/jars
[2024-05-18T23:21:00.754+0000] {spark_submit.py:571} INFO - org.apache.spark#spark-sql-kafka-0-10_2.12 added as a dependency
[2024-05-18T23:21:00.755+0000] {spark_submit.py:571} INFO - com.datastax.spark#spark-cassandra-connector_2.12 added as a dependency
[2024-05-18T23:21:00.756+0000] {spark_submit.py:571} INFO - org.apache.kafka#kafka-clients added as a dependency
[2024-05-18T23:21:00.756+0000] {spark_submit.py:571} INFO - commons-logging#commons-logging added as a dependency
[2024-05-18T23:21:00.756+0000] {spark_submit.py:571} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-984573c0-b0c3-408d-85bc-10920a2990e7;1.0
[2024-05-18T23:21:00.757+0000] {spark_submit.py:571} INFO - confs: [default]
[2024-05-18T23:21:02.519+0000] {spark_submit.py:571} INFO - found org.apache.spark#spark-sql-kafka-0-10_2.12;3.5.1 in central
[2024-05-18T23:21:02.817+0000] {spark_submit.py:571} INFO - found org.apache.spark#spark-token-provider-kafka-0-10_2.12;3.5.1 in central
[2024-05-18T23:21:03.400+0000] {spark_submit.py:571} INFO - found org.apache.hadoop#hadoop-client-runtime;3.3.4 in central
[2024-05-18T23:21:03.819+0000] {spark_submit.py:571} INFO - found org.apache.hadoop#hadoop-client-api;3.3.4 in central
[2024-05-18T23:21:03.975+0000] {spark_submit.py:571} INFO - found org.xerial.snappy#snappy-java;1.1.10.3 in central
[2024-05-18T23:21:04.586+0000] {spark_submit.py:571} INFO - found org.slf4j#slf4j-api;2.0.7 in central
[2024-05-18T23:21:05.095+0000] {spark_submit.py:571} INFO - found com.google.code.findbugs#jsr305;3.0.0 in central
[2024-05-18T23:21:06.049+0000] {spark_submit.py:571} INFO - found org.apache.commons#commons-pool2;2.11.1 in central
[2024-05-18T23:21:06.249+0000] {spark_submit.py:571} INFO - found com.datastax.spark#spark-cassandra-connector_2.12;3.5.0 in central
[2024-05-18T23:21:06.343+0000] {spark_submit.py:571} INFO - found com.datastax.spark#spark-cassandra-connector-driver_2.12;3.5.0 in central
[2024-05-18T23:21:06.416+0000] {spark_submit.py:571} INFO - found org.scala-lang.modules#scala-collection-compat_2.12;2.11.0 in central
[2024-05-18T23:21:07.549+0000] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-05-18T23:21:07.560+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-18T23:21:07.569+0000] {process_utils.py:132} INFO - Sending 15 to group 201. PIDs of all processes in the group: [203, 201]
[2024-05-18T23:21:07.570+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 201
[2024-05-18T23:21:07.573+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-05-18T23:21:07.583+0000] {spark_submit.py:697} INFO - Sending kill signal to spark-submit
[2024-05-18T23:21:07.589+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T23:21:07.759+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=203, status='terminated', started='23:20:57') (203) terminated with exit code None
[2024-05-18T23:21:07.761+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=201, status='terminated', exitcode=0, started='23:20:56') (201) terminated with exit code 0
